### DISCUSSION TITLE
Could not log in An unhandled error occurred while serving the request

### DISCUSSION THREAD

**Discussion by Oscar-Santacruz** [datalust/seq-tickets#2249] [Help]
Could not log in
An unhandled error occurred while serving the request (token: 0a2d1c46ef3c46d1b0347984b7987c40).

I have a recurring problem with the seq deployed in mini kubernetes which is this error mentioned, I usually restart my pod so that I can log in again

I have read about viewing the logs and diagnostics with the admin but it happens that I have the admin user and I cannot log in

**Comment by nblumhardt** [comment]
Hi @Oscar-Santacruz,

Something like:

```
kubectl logs pod seq
```

will give you more info to work with. Hope this helps!

Nick
